package.json

```diff
@@ -36,9 +36,9 @@
     "fixturify-project": "^2.1.0",
     "jest": "^26.4.2",
     "prettier": "^2.1.1",
-    "release-it": "^13.7.0",
-    "release-it-lerna-changelog": "^2.3.0",
```

Comment:
@rwjblue i think the thing that's needed here is for the transitive deps to `release-it` to be de-duplicatable whenever bumping majors of `release-it`